File: e2e/selenide-junit5-selenium-grid/src/test/java/test/desktop/taskInfo/TaskInfoTest.java
package test.desktop.taskInfo;

import io.qameta.allure.Epic;
import io.qameta.allure.Feature;
import io.qameta.allure.TmsLink;
import test.base.DesktopTest;
import test.spec.taskInfo.ITaskInfoTest;

import static data.AllureEpic.TASK_MANAGEMENT;

@Epic(TASK_MANAGEMENT)
@Feature("View task info")
@TmsLink("102")
class TaskInfoTest extends DesktopTest implements ITaskInfoTest {}

File: e2e/maestro/tests/task-info/task-info.test.ts
import { AllureEpic } from '@/data/AllureEpic';
import { maestroTestCase } from '@/runner/maestroTestCase';

export default maestroTestCase(import.meta.url, {
  name: 'should display task details when info opened',
  epic: AllureEpic.TASK_MANAGEMENT,
  feature: 'View task info',
  tms: '102',
  suite: 'mocked',
  setupMocks: async (support, context) => {
    const response = support.mock.api.fromResponse(context.createTaskResponse());

    await support.reset();
    await support.mock.api.getTasks(response);
    await support.mock.api.getTasks(response);
    await support.mock.api.getTask(response);
    await support.mock.api.getIsValid(true);

    return context.toMaestroEnv();
  },
});
